### PR TITLE
Disable search box autofocus

### PR DESCRIPTION
**Pull request by silverwind**
Various pages like the issue list autofocus their search box which I find rather intrusive because more often than not I don't want to type into the search box, it's distracting if the focus jumps after page load. Disable this behaviour.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

File: templates/explore/code.tmpl [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		<form class="ui form ignore-dirty" style="max-width: 100%">
             <input type="hidden" name="tab" value="{{$.TabName}}">
             <div class="ui fluid action input">
-                <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+                <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```
Comment (lunny) [inline templates/explore/code.tmpl]: Didn't this is a real search page? Maybe we should keep it on this file.

File: templates/explore/search.tmpl [hunk 2]
```diff
@@ -18,7 +18,7 @@
 <form class="ui form ignore-dirty" style="max-width: 90%">
 	<input type="hidden" name="tab" value="{{$.TabName}}">
 	<div class="ui fluid action input">
-	  <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+	  <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```
Comment (lunny) [inline templates/explore/search.tmpl]: The same as the above.

File: templates/repo/issue/search.tmpl [hunk 3]
```diff
@@ -5,7 +5,7 @@
 		<input type="hidden" name="labels" value="{{.SelectLabels}}"/>
 		<input type="hidden" name="milestone" value="{{$.MilestoneID}}"/>
 		<input type="hidden" name="assignee" value="{{$.AssigneeID}}"/>
-		<input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+		<input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```
Comment (lunny) [inline templates/repo/issue/search.tmpl:8]: ~The same as the above.~

**Review by lunny — CHANGES_REQUESTED** [review]

File: templates/explore/code.tmpl (hunk 1 above)
Comment (silverwind) [reply templates/explore/code.tmpl]: I guess we can keep it on the explore pages, I just want to eliminate it on the issue/pr pages.

File: templates/explore/search.tmpl (hunk 2 above)
Comment (silverwind) [reply templates/explore/search.tmpl:21]: This sub-template is used on the issue list where I do want to remove it.

File: templates/repo/issue/search.tmpl (hunk 3 above)
Comment (silverwind) [reply templates/repo/issue/search.tmpl:8]: This subtemplate is used on the issue list where I certainly want to remove it.

Comment (lunny) [reply templates/repo/issue/search.tmpl:8]: OK. I mean all the explore pages could keep the autofocus

**Comment by silverwind** [comment]
@lunny updated to only disable on issue/pr pages.

**Review by lunny — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**